manifests/init.pp

```diff
@@ -283,171 +232,123 @@
+  Boolean $oauth_enabled = $::pulp::params::oauth_enabled,
+  String $oauth_key = $::pulp::params::oauth_key,
+  String $oauth_secret = $::pulp::params::oauth_secret,
+  Integer[0] $max_keep_alive = $::pulp::params::max_keep_alive,
+  String $messaging_url = $::pulp::params::messaging_url,
+  Enum['qpid', 'rabbitmq'] $messaging_transport = $::pulp::params::messaging_transport,
+  Boolean $messaging_auth_enabled = $::pulp::params::messaging_auth_enabled,
+  Optional[Stdlib::Absolutepath] $messaging_ca_cert = $::pulp::params::messaging_ca_cert,
+  Optional[Stdlib::Absolutepath] $messaging_client_cert = $::pulp::params::messaging_client_cert,
+  String $messaging_topic_exchange = $::pulp::params::messaging_topic_exchange,
+  Boolean $messaging_event_notifications_enabled = $::pulp::params::messaging_event_notifications_enabled,
+  Optional[String] $messaging_event_notification_url = $::pulp::params::messaging_event_notification_url,
+  String $messaging_version = $::pulp::params::messaging_version,
+  String $broker_url = $::pulp::params::broker_url,
+  Boolean $broker_use_ssl = $::pulp::params::broker_use_ssl,
+  Optional[String] $tasks_login_method = $::pulp::params::tasks_login_method,
+  String $email_host = $::pulp::params::email_host,
+  Integer $email_port = $::pulp::params::email_port,
+  String $email_from = $::pulp::params::email_from,
+  Boolean $email_enabled = $::pulp::params::email_enabled,
+  Boolean $manage_squid = $::pulp::params::manage_squid,
+  Optional[String] $lazy_redirect_host = $::pulp::params::lazy_redirect_host,
+  Optional[Integer[0, 65535]] $lazy_redirect_port = $::pulp::params::lazy_redirect_port,
+  Optional[String] $lazy_redirect_path = $::pulp::params::lazy_redirect_path,
+  Boolean $lazy_https_retrieval = $::pulp::params::lazy_https_retrieval,
+  Integer[0] $lazy_download_interval = $::pulp::params::lazy_download_interval,
+  Integer[0] $lazy_download_concurrency = $::pulp::params::lazy_download_concurrency,
+  Optional[Stdlib::Absolutepath] $consumers_crl = $::pulp::params::consumers_crl,
+  Boolean $reset_cache = $::pulp::params::reset_cache,
+  Enum['none', 'optional', 'require', 'optional_no_ca'] $ssl_verify_client = $::pulp::params::ssl_verify_client,
+  Variant[Array[String], String] $ssl_protocol = $::pulp::params::ssl_protocol,
+  Boolean $repo_auth = $::pulp::params::repo_auth,
+  Optional[String] $proxy_url = $::pulp::params::proxy_url,
+  Optional[Integer[0, 65535]] $proxy_port = $::pulp::params::proxy_port,
+  Optional[String] $proxy_username = $::pulp::params::proxy_username,
+  Optional[String] $proxy_password = $::pulp::params::proxy_password,
+  Optional[String] $yum_max_speed = $::pulp::params::yum_max_speed,
+  Integer[0] $num_workers = $::pulp::params::num_workers,
+  Boolean $enable_katello = $::pulp::params::enable_katello,
+  Boolean $enable_crane = $::pulp::params::enable_crane,
+  Optional[Integer[0]] $max_tasks_per_child = $::pulp::params::max_tasks_per_child,
```

Comment:
https://github.com/Katello/puppet-pulp/blob/74fb5e2c9b666253ae0bc3b4465f2344b8f235d2/manifests/params.pp#L111 is there, right?